src/index.js

```diff
@@ -0,0 +1,14 @@
+import './style.css';
+
+// const recentScore = document.getElementsById('recent')
+const leaderboardForm = document.getElementsById('add-your-score');
+
+leaderboardForm.addEventListener('submit', async (e) => {
+  e.preventDefault();
+  // const person = document.getElementById('input-name');
+  // const score = document.getElementById('input-score');
+  // const leaderboardFormData = {
+  //   person: person.value,
+  //   score: parseInt(score.value, 10),
+  // };
+});
```

Comment:
- [ ] Please delete commented code for follow js best practices. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20old%20pieces%20of%20code%20as%20inline%20comments.%20They%20will%20make%20your%20project%20look%20messy.%20If%20you%20need%20to%20review%20a%20previous%20version%20of%20your%20code%2C%20you%20can%20always%20use%20git%20history.)

Reply:
@ahmetbozaci Thank you so much